modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -1869,6 +1869,7 @@ export const console: ConsoleNS = {
                                 hint: {
                                     customOidc: "මෙම සහතිකය සත්‍යාපනයෙන් පසු ලැබුණු <1>id_token</1> සංකේතනය " +
                                         "කිරීමට භාවිතා කරයි.",
+                                    customPassiveSTS: "මෙම සහතිකය request තහවුරු කිරීම සඳහා භාවිතා කරයි.",
```

Comment:
```suggestion
                                    customPassiveSTS: "මෙම සහතිකය ඉල්ලීම තහවුරු කිරීම සඳහා භාවිතා කරයි.",
```

Reply:
`ඉල්ලීම` seemed a bit wrong in this context 😁. Any other suggestions to convert `request payload`?